README.md

```diff
@@ -61,6 +61,22 @@ Pull the docker image from `qwantresearch/erdapfel` [![Docker Pulls](https://img
 
 [![Travis Build](https://travis-ci.org/QwantResearch/erdapfel.svg?branch=master)](https://travis-ci.org/QwantResearch/erdapfel)
 
+## Using your own map tiles server
+
+We explained earlier how to override some variables using `TILEVIEW_`. If you want to use your own map tiles server(s), you can do it by running with:
+
+```bash
+> TILEVIEW_mapStyle_baseMapUrl='["http://your-server/ozbasemap/{z}/{x}/{y}.pbf"]' \
+TILEVIEW_mapStyle_poiMapUrl='["http://your-server/ozpoi/{z}/{x}/{y}.pbf"]' \
+npm start
+```
+
+The part `{z}/{x}/{y}.pbf` is mandatory to allow `erdapfel` how to set the position and zoom.
+
+The `ozbasemap` and `ozpoi` are paths to where interrogate the server to get map's and POIs' tiles (so it might be different in your case).
```

Comment:
Where this `oz` comes from?
I would just ditch this paragraph and replace `ozbasemap` with `path-to-basemap-tiles` and `ozpoi` by `path-to-poi-tiles`, or something generic like that.